src/DataStructures/Tensor/Expressions/AddSubtract.hpp

```diff
@@ -353,76 +368,94 @@ struct AddSub<T1, T2, ArgsList1<Args1...>, ArgsList2<Args2...>, Sign>
   static constexpr size_t num_ops_subtree =
       num_ops_left_child + num_ops_right_child + 1;
 
+  // === Properties for splitting up subexpressions along the primary path ===
+  // These defintiions only have meaning if this expression actually ends up
```

Comment:
typo: definitions

Reply:
Fixed this in the other files that has this typo, as well